### DISCUSSION TITLE
这个forge怎么使用？付费后网址打不开

### DISCUSSION THREAD

**Discussion by O2isadrug** [Meloong-Git/PCL#2592] [提问] [answered]
再这条帖子下付的费 https://afdian.net/p/20f32c6c67e211e9b8d952540025c377
跳转至 这个打不开的页面 https://pcl2-1253424809.cos.ap-chengdu.myqcloud.com/Forge/Download.exe

我的邮箱是 1936002817@qq.com

麻烦教我一下如何使用 或者 发个教程链接

![29d9fbe7160f00d5815e2cf24a6daea](https://github.com/Hex-Dragon/PCL2/assets/46131479/af6e320c-211e-4256-9acd-5a59063d7a80)

**Comment by WForst-Breeze** [comment, marked as answer]
如果你要安装 Forge，请在下载页面选择 Minecraft 版本后，在弹出的窗口中点击 Forge 下拉框并选择您要安装的 Forge 版本即可。

如您需要详细教程，打开 PCL2 前往 `更多`→`帮助`→`Minecraft`→`安装 Mod`→`安装 Mod 加载器-Forge` 中查看逐步教程

**Comment by hejiehao** [comment]
~~不会为了安装Forge氪了￥2.33吧~~

**Reply by MoYuan-CN** [reply]
真的.png

**Comment by so1ve** [comment]
pcl自带forge下载安装…
